packages/components/src/utils/card-style.ts

```diff
@@ -0,0 +1,51 @@
+import { Card } from '@models/Card';
+import { isPlayingCard } from '@models/PlayingCard';
+import { isUnoCard } from '@models/UnoCard';
+import { PlayingCardDimensions } from '@atoms/SvgPlayingCard';
+import { UnoCardDimensions } from '@atoms/UnoCardFace';
+import { Dimensions } from '@models/Dimensions';
+import { calculateCardDimensions } from './card-dimensions';
+
+export interface CardTypeStyle {
+  /** Default card height - used to calculate card ratio */
+  dimensions: Dimensions;
+  /** Card border radius */
+  borderRadius?: number;
+}
+
+export type CustomCardSytleFn = (card: Card, currentStyle?: CardTypeStyle) => CardTypeStyle;
+
+/**
+ * Returns the style for a given card type
+ * @param cards the card that configures the type
+ * @param tableDimentions the dimensions of the table
+ * @param customCardStyle a custom fn for style
+ */
+export const getCardStyleFn = (
```

Comment:
Why puttin the suffix `Fn` to the function name? We already know it's a function 😄

Reply:
Changed to Factory